src/main/resources/application.yml

```
@@ -0,0 +1,8 @@
+spring:
+  profiles:
+    active: dev
+file:
+  voucher:
+    path: ./file/voucher_list.csv
+  customer:
+    path: ./file/customer_blacklist.csv
```

현재는 무조건 dev 프로필로만 실행되고 있습니다.
하지만 local 프로필의 경우엔 바우처는 file을 사용하지 않기 때문에 아래처럼 프로필 분리가 필요할 것 같습니다

```suggestion
spring:
  profiles:
    default: local
file:
  customer:
    path: ./file/customer_blacklist.csv

---

spring:
  config:
    activate:
      on-profile: local

---

spring:
  config:
    activate:
      on-profile: dev
file:
  voucher:
    path: ./file/voucher_list.csv
```